__tests__/ExpensiMark-Markdown-Truncate-test.js

```diff
@@ -0,0 +1,101 @@
+import ExpensiMark from '../lib/ExpensiMark';
+
+const parser = new ExpensiMark();
+
+describe('truncateHTML', () => {
+    test('should return original text as HTML if it does not exceed the limit', () => {
+        const markdown = 'This is a *short* text that does not exceed the character limit.';
+        const limit = 100;
+        const html = parser.replace(markdown);
+        const result = parser.truncateHTML(html, limit, {ellipsis: true});
+        expect(result).toBe(html);
+    });
+
+    test('should truncate HTML and add ellipsis if it exceeds the limit', () => {
+        const markdown =
+            'This is a *long* text that exceeds the character limit. It contains multiple sentences to test the truncation functionality. The truncation should occur at the nearest space to avoid cutting off words.';
```

Comment:
This blurb doesn't match the expected result, since we truncate mid-word with ellipses, not at the nearest space

Reply:
Can we update this copy then since it's incorrect? I know the test title is correct but it's good to be consistent still